src/blog/v8-release-80.md

```diff
@@ -0,0 +1,136 @@
8.4">See Thibault Molleman&apos;s other Tweets</tspan></text><path fill="url(#e)" d="M797.3 318.3h17.5v17.5h-17.5z"/><path class="divider" d="M336.8 307.6h498"/></a></g></g><g data-id="twitter-widget"><path class="bg" d="M522 272.1l400-7 2.6 155.4-400 7z"/><g transform="rotate(-1 4619.2 -7976.5) scale(.8)"><a><image width="36" height="36" x="494.3" y="270.2" href="https://pbs.twimg.com/profile_images/1185226911221452802/nH7ykGFP_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="539.3" y="280.8">Greg Miernicki aka 🅖🅡🅔🅖</tspan></text><text x="66" y="40" class="subText"><tspan x="539.3" y="297.6">@gregulatore</tspan></text></a><a><path fill="url(#a)" d="M932.3 270.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="494.3" y="330">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="494.3" y="355.2">Anything special planned for v8 v8.0?</tspan></text><image width="22.4" height="20" x="764.7" y="341.2" href="https://abs.twimg.com/emoji/v2/72x72/1f605.png"/></g><a class="like"><path d="M494.3 374.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="514.8" y="385.2">5</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="534.6" y="385.2">16:43 - 13 Aug 2019</tspan></text></a><a><path fill="url(#c)" d="M934.3 374.4h18v18h-18z"/></a><a><path fill="url(#d)" d="M494.3 415.2h17.5v17.5h-17.5z"/><text x="43" y="165" class="seeOther"><tspan x="515.8" y="425.2">See Greg Miernicki aka 🅖🅡🅔🅖&apos;s other Tweets</tspan></text><path fill="url(#e)" d="M934.8 415.2h17.5v17.5h-17.5z"/><path class="divider" d="M474.3 404.4h498"/></a></g></g><g data-id="twitter-widget"><path class="bg" d="M671.2 141.3l394 69.5-30 169.7-394-69.5z"/><g transform="rotate(10 469.6 1210) scale(.8)"><a><image width="36" height="36" x="624.2" y="174.2" href="https://pbs.twimg.com/profile_images/22070422/south_park_me__art_raged__normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="669.2" y="184.8">SignpostMarv</tspan></text><text x="66" y="40" class="subText"><tspan x="669.2" y="201.6">@SignpostMarv</tspan></text></a><a><path fill="url(#a)" d="M1062.2 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="624.2" y="234">Replying to @v8js @ChromiumDev</tspan></text></a><text x="21" y="92"><tspan x="624.2" y="258.4">are you going to be having an extra special party when V8 goes</tspan><tspan x="624.2" y="279.4">v8 ?</tspan></text><a class="like"><path d="M624.2 299.6h17.5v17.5h-17.5z"/><text x="42" y="146"><tspan x="644.7" y="310.4">18</tspan></text></a><a><text x="69" y="147" class="subText"><tspan x="672.3" y="310.4">16:20 - 27 Sep 2019</tspan></text></a><a><path fill="url(#c)" d="M1064.2 299.6h18v18h-18z"/></a><a><path fill="url(#d)" d="M624.2 340.3h17.5v17.5h-17.5z"/><text x="43" y="186" class="seeOther"><tspan x="645.7" y="350.4">See SignpostMarv&apos;s other Tweets</tspan></text><path fill="url(#e)" d="M1064.7 340.3h17.5v17.5h-17.5z"/><path class="divider" d="M604.2 329.6h498"/></a></g></g></svg>
```

Comment:
Removed unnecessary escaping in the data URLs:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="1075" height="290" viewBox="-5 140 1075 290" style="position:relative;max-width:90vw;height:auto;margin-top:max(-47px,-5vw);left:50%;margin-left:max(-547px,-45vw)"><style>text{font-family:Helvetica,Roboto,&quot;Segoe UI&quot;,Calibri,sans-serif;fill:#1c2022;font-weight:400}.bg,.divider{stroke:#e1e8ed;stroke-width:.8;fill:#fff}a.name text{font-weight:700}.seeOther,.subText,a.like text,a.name .subText{fill:#697882;font-size:14px;font-weight:400}.seeOther{fill:#2b7bb9}a.like path{fill:url(#b)}a:hover text{fill:#3b94d9}a.like:hover text{fill:#e0245e}a.like:hover path{fill:url(#B)}.dark .bg{stroke:#66757f;fill:#000}.dark text{fill:#f5f8fa}.dark .subText,.dark a.name .subText,.dark a.like text{fill:#8899a6}.dark .seeOther{fill:#3b94d9}.dark a:hover text{fill:#55acee}.dark a.like:hover text{fill:#e0245e}</style><defs><pattern id="a" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 72 72%22><path fill=%22none%22 d=%22M0 0h72v72H0z%22/><path class=%22icon%22 fill=%22%231da1f2%22 d=%22M68.812 15.14c-2.348 1.04-4.87 1.744-7.52 2.06 2.704-1.62 4.78-4.186 5.757-7.243-2.53 1.5-5.33 2.592-8.314 3.176C56.35 10.59 52.948 9 49.182 9c-7.23 0-13.092 5.86-13.092 13.093 0 1.026.118 2.02.338 2.98C25.543 24.527 15.9 19.318 9.44 11.396c-1.125 1.936-1.77 4.184-1.77 6.58 0 4.543 2.312 8.552 5.824 10.9-2.146-.07-4.165-.658-5.93-1.64-.002.056-.002.11-.002.163 0 6.345 4.513 11.638 10.504 12.84-1.1.298-2.256.457-3.45.457-.845 0-1.666-.078-2.464-.23 1.667 5.2 6.5 8.985 12.23 9.09-4.482 3.51-10.13 5.605-16.26 5.605-1.055 0-2.096-.06-3.122-.184 5.794 3.717 12.676 5.882 20.067 5.882 24.083 0 37.25-19.95 37.25-37.25 0-.565-.013-1.133-.038-1.693 2.558-1.847 4.778-4.15 6.532-6.774z%22/></svg>"/></pattern><pattern id="b" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23697882%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern><pattern id="c" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23697882%22 d=%22M12 18.042c-.553 0-1-.447-1-1v-5.5c0-.553.447-1 1-1s1 .447 1 1v5.5c0 .553-.447 1-1 1z%22/><circle class=%22icon%22 fill=%22%23697882%22 cx=%2212%22 cy=%228.042%22 r=%221.25%22/><path class=%22icon%22 fill=%22%23697882%22 d=%22M12 22.75C6.072 22.75 1.25 17.928 1.25 12S6.072 1.25 12 1.25 22.75 6.072 22.75 12 17.928 22.75 12 22.75zm0-20C6.9 2.75 2.75 6.9 2.75 12S6.9 21.25 12 21.25s9.25-4.15 9.25-9.25S17.1 2.75 12 2.75z%22/></svg>"/></pattern><pattern id="d" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%232b7bb9%22 d=%22M12 11.816c1.355 0 2.872-.15 3.84-1.256.814-.93 1.078-2.368.806-4.392-.38-2.825-2.117-4.512-4.646-4.512S7.734 3.343 7.354 6.17c-.272 2.022-.008 3.46.806 4.39.968 1.107 2.485 1.256 3.84 1.256zM8.84 6.368c.162-1.2.787-3.212 3.16-3.212s2.998 2.013 3.16 3.212c.207 1.55.057 2.627-.45 3.205-.455.52-1.266.743-2.71.743s-2.255-.223-2.71-.743c-.507-.578-.657-1.656-.45-3.205zm11.44 12.868c-.877-3.526-4.282-5.99-8.28-5.99s-7.403 2.464-8.28 5.99c-.172.692-.028 1.4.395 1.94.408.52 1.04.82 1.733.82h12.304c.693 0 1.325-.3 1.733-.82.424-.54.567-1.247.394-1.94zm-1.576 1.016c-.126.16-.316.246-.552.246H5.848c-.235 0-.426-.085-.552-.246-.137-.174-.18-.412-.12-.654.71-2.855 3.517-4.85 6.824-4.85s6.114 1.994 6.824 4.85c.06.242.017.48-.12.654z%22/></svg>"/></pattern><pattern id="e" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23697882%22 d=%22M17.207 11.293l-7.5-7.5c-.39-.39-1.023-.39-1.414 0s-.39 1.023 0 1.414L15.086 12l-6.793 6.793c-.39.39-.39 1.023 0 1.414.195.195.45.293.707.293s.512-.098.707-.293l7.5-7.5c.39-.39.39-1.023 0-1.414z%22/></svg>"/></pattern><pattern id="B" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23E0245E%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern></defs><g data-id="twitter-widget"><path class="bg" d="M-2.2 222.4l398.4-34.8 13.6 155.3-398.4 34.8z"/><g transform="rotate(-5 830.8 -212.3) scale(.8)"><a><image width="36" height="36" x="-25.2" y="206.2" href="https://pbs.twimg.com/profile_images/1204331159582367745/YF3T9NOI_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="19.8" y="218.6">Josebaba</tspan></text><image width="22.4" height="20" x="93.6" y="206.2" href="http://abs.twimg.com/emoji/v2/72x72/1f4a5.png"/><text x="66" y="42" class="subText"><tspan x="19.8" y="235.4">@fullstackmofo</tspan></text></a><a><path fill="url(#a)" d="M412.8 206.2h20v20h-20z"/></a><a><text x="21" y="72" class="subText"><tspan x="-25.2" y="266.8">Replying to @v8js</tspan></text></a><text x="21" y="93"><tspan x="-25.2" y="291.2">V8 almost at v8</tspan></text><a class="like"><path d="M-25.2 310.4h17.5v17.5h-17.5z"/><text x="42" y="125" class="subText"><tspan x="-4.7" y="321.2">4</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="15.1" y="321.2">22:09 - 20 Nov 2019</tspan></text></a><a><path fill="url(#c)" d="M414.8 310.4h18v18h-18z"/></a><a><path fill="url(#d)" d="M-25.2 351.5h17.5V369h-17.5z"/><g class="seeOther"><text x="43" y="165"><tspan x="-3.7" y="361.9">See Josebaba</tspan></text><image width="19.6" height="17.5" x="85.8" y="350.4" href="http://abs.twimg.com/emoji/v2/72x72/1f4a5.png"/><text x="43" y="165"><tspan x="105.4" y="361.9">’s other Tweets</tspan></text></g><path fill="url(#e)" d="M415.3 351.5h17.5V369h-17.5z"/><path class="divider" d="M-45.2 340.4h498"/></a></g></g><g data-id="twitter-widget"><path class="bg" d="M147.2 238.9l399 27.9-10.8 155-399-28z"/><g transform="rotate(4 -638.7 1274.7) scale(.8)"><a><image width="36" height="36" x="112.3" y="254.2" href="https://pbs.twimg.com/profile_images/1205935667596943360/vN1fmH58_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="157.3" y="264.8">Connor ’Stryxus’ Shearer</tspan></text><text x="66" y="40" class="subText"><tspan x="157.3" y="281.6">@Stryxus</tspan></text></a><a><path fill="url(#a)" d="M550.3 254.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="112.3" y="314">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="112.3" y="339.2">What happens when v8 reaches v8?</tspan></text><image width="22.4" height="20" x="375.6" y="325.2" href="https://abs.twimg.com/emoji/v2/72x72/1f914.png"/></g><a class="like"><path d="M112.3 358.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="132.8" y="369.2">11</tspan></text></a><a><text x="68" y="126" class="subText"><tspan x="159.4" y="369.2">17:19 - 20 Nov 2019</tspan></text></a><a><path fill="url(#c)" d="M552.3 358.4h18v18h-18z"/></a><a><path fill="url(#d)" d="M112.3 399.2h17.5v17.5h-17.5z"/><text x="43" y="165" class="seeOther"><tspan x="133.8" y="409.2">See Connor ’Stryxus’ Shearer’s other Tweets</tspan></text><path fill="url(#e)" d="M552.8 399.2h17.5v17.5h-17.5z"/><path class="divider" d="M92.3 388.4h498"/></a></g></g><g data-id="twitter-widget"><path class="bg" d="M383.2 179.6l399.8-14 5.4 154.6-399.8 14z"/><g transform="rotate(-2 1958.9 -3131) scale(.8)"><a><image width="36" height="36" x="356.8" y="174.2" href="https://pbs.twimg.com/profile_images/977905767415205888/Pr5NSZ_T_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="401.8" y="184.8">Thibault Molleman</tspan></text><text x="66" y="40" class="subText"><tspan x="401.8" y="201.6">@thibaultmol</tspan></text></a><a><path fill="url(#a)" d="M794.8 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="356.8" y="234">Replying to @v8js</tspan></text></a><text x="21" y="92"><tspan x="356.8" y="258.4">Wait. What happens when we get V8 V8?</tspan></text><a class="like"><path d="M356.8 277.6h17.5v17.5h-17.5z"/></a><a><text x="54" y="125" class="subText"><tspan x="389.3" y="288.4">11:37 - 20 Jun 2019</tspan></text></a><a><path fill="url(#c)" d="M796.8 277.6h18v18h-18z"/></a><a><path fill="url(#d)" d="M356.8 318.3h17.5v17.5h-17.5z"/><text x="43" y="164" class="seeOther"><tspan x="378.3" y="328.4">See Thibault Molleman’s other Tweets</tspan></text><path fill="url(#e)" d="M797.3 318.3h17.5v17.5h-17.5z"/><path class="divider" d="M336.8 307.6h498"/></a></g></g><g data-id="twitter-widget"><path class="bg" d="M522 272.1l400-7 2.6 155.4-400 7z"/><g transform="rotate(-1 4619.2 -7976.5) scale(.8)"><a><image width="36" height="36" x="494.3" y="270.2" href="https://pbs.twimg.com/profile_images/1185226911221452802/nH7ykGFP_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="539.3" y="280.8">Greg Miernicki aka 🅖🅡🅔🅖</tspan></text><text x="66" y="40" class="subText"><tspan x="539.3" y="297.6">@gregulatore</tspan></text></a><a><path fill="url(#a)" d="M932.3 270.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="494.3" y="330">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="494.3" y="355.2">Anything special planned for v8 v8.0?</tspan></text><image width="22.4" height="20" x="764.7" y="341.2" href="https://abs.twimg.com/emoji/v2/72x72/1f605.png"/></g><a class="like"><path d="M494.3 374.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="514.8" y="385.2">5</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="534.6" y="385.2">16:43 - 13 Aug 2019</tspan></text></a><a><path fill="url(#c)" d="M934.3 374.4h18v18h-18z"/></a><a><path fill="url(#d)" d="M494.3 415.2h17.5v17.5h-17.5z"/><text x="43" y="165" class="seeOther"><tspan x="515.8" y="425.2">See Greg Miernicki aka 🅖🅡🅔🅖’s other Tweets</tspan></text><path fill="url(#e)" d="M934.8 415.2h17.5v17.5h-17.5z"/><path class="divider" d="M474.3 404.4h498"/></a></g></g><g data-id="twitter-widget"><path class="bg" d="M671.2 141.3l394 69.5-30 169.7-394-69.5z"/><g transform="rotate(10 469.6 1210) scale(.8)"><a><image width="36" height="36" x="624.2" y="174.2" href="https://pbs.twimg.com/profile_images/22070422/south_park_me__art_raged__normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="669.2" y="184.8">SignpostMarv</tspan></text><text x="66" y="40" class="subText"><tspan x="669.2" y="201.6">@SignpostMarv</tspan></text></a><a><path fill="url(#a)" d="M1062.2 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="624.2" y="234">Replying to @v8js @ChromiumDev</tspan></text></a><text x="21" y="92"><tspan x="624.2" y="258.4">are you going to be having an extra special party when V8 goes</tspan><tspan x="624.2" y="279.4">v8 ?</tspan></text><a class="like"><path d="M624.2 299.6h17.5v17.5h-17.5z"/><text x="42" y="146"><tspan x="644.7" y="310.4">18</tspan></text></a><a><text x="69" y="147" class="subText"><tspan x="672.3" y="310.4">16:20 - 27 Sep 2019</tspan></text></a><a><path fill="url(#c)" d="M1064.2 299.6h18v18h-18z"/></a><a><path fill="url(#d)" d="M624.2 340.3h17.5v17.5h-17.5z"/><text x="43" y="186" class="seeOther"><tspan x="645.7" y="350.4">See SignpostMarv’s other Tweets</tspan></text><path fill="url(#e)" d="M1064.7 340.3h17.5v17.5h-17.5z"/><path class="divider" d="M604.2 329.6h498"/></a></g></g></svg>
```

Reply:
Improved:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="1075" height="290" viewBox="-5 140 1075 290" style="position:relative;max-width:90vw;height:auto;margin-top:max(-47px,-5vw);left:50%;margin-left:max(-547px,-45vw)"><style>text{font-family:Helvetica,Roboto,&quot;Segoe UI&quot;,Calibri,sans-serif;fill:#1c2022;font-weight:400}.bg,.divider{stroke:#e1e8ed;stroke-width:.8;fill:#fff}a.name text{font-weight:700}.seeOther,.subText,a.like text,a.name .subText{fill:#697882;font-size:14px;font-weight:400}.seeOther{fill:#2b7bb9}a.like path{fill:url(#b)}a:hover text{fill:#3b94d9}a.like:hover text{fill:#e0245e}a.like:hover path{fill:url(#B)}.dark .bg{stroke:#66757f;fill:#000}.dark text{fill:#f5f8fa}.dark .subText,.dark a.name .subText,.dark a.like text{fill:#8899a6}.dark .seeOther{fill:#3b94d9}.dark a:hover text{fill:#55acee}.dark a.like:hover text{fill:#e0245e}</style><defs><pattern id="a" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 72 72%22><path fill=%22none%22 d=%22M0 0h72v72H0z%22/><path class=%22icon%22 fill=%22%231da1f2%22 d=%22M68.812 15.14c-2.348 1.04-4.87 1.744-7.52 2.06 2.704-1.62 4.78-4.186 5.757-7.243-2.53 1.5-5.33 2.592-8.314 3.176C56.35 10.59 52.948 9 49.182 9c-7.23 0-13.092 5.86-13.092 13.093 0 1.026.118 2.02.338 2.98C25.543 24.527 15.9 19.318 9.44 11.396c-1.125 1.936-1.77 4.184-1.77 6.58 0 4.543 2.312 8.552 5.824 10.9-2.146-.07-4.165-.658-5.93-1.64-.002.056-.002.11-.002.163 0 6.345 4.513 11.638 10.504 12.84-1.1.298-2.256.457-3.45.457-.845 0-1.666-.078-2.464-.23 1.667 5.2 6.5 8.985 12.23 9.09-4.482 3.51-10.13 5.605-16.26 5.605-1.055 0-2.096-.06-3.122-.184 5.794 3.717 12.676 5.882 20.067 5.882 24.083 0 37.25-19.95 37.25-37.25 0-.565-.013-1.133-.038-1.693 2.558-1.847 4.778-4.15 6.532-6.774z%22/></svg>"/></pattern><pattern id="b" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23697882%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern><pattern id="B" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23E0245E%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern></defs><g data-id="twitter-widget"><path class="bg" d="M-2.2 222.4l398.4-34.8 13.6 127.3-398.4 34.8z"/><g transform="rotate(-5 830.8 -212.3) scale(.8)"><a><image width="36" height="36" x="-25.2" y="206.2" href="https://pbs.twimg.com/profile_images/1204331159582367745/YF3T9NOI_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="19.8" y="218.6">Josebaba</tspan></text><image width="22.4" height="20" x="93.6" y="206.2" href="http://abs.twimg.com/emoji/v2/72x72/1f4a5.png"/><text x="66" y="42" class="subText"><tspan x="19.8" y="235.4">@fullstackmofo</tspan></text></a><a><path fill="url(#a)" d="M412.8 206.2h20v20h-20z"/></a><a><text x="21" y="72" class="subText"><tspan x="-25.2" y="266.8">Replying to @v8js</tspan></text></a><text x="21" y="93"><tspan x="-25.2" y="291.2">V8 almost at v8</tspan></text><a class="like"><path d="M-25.2 307.4h17.5v17.5h-17.5z"/><text x="42" y="125" class="subText"><tspan x="-4.7" y="321.2">4</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="15.1" y="321.2">22:09 - 20 Nov 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M147.2 238.9l399 27.9-10.8 127-399-28z"/><g transform="rotate(4 -638.7 1274.7) scale(.8)"><a><image width="36" height="36" x="112.3" y="254.2" href="https://pbs.twimg.com/profile_images/1205935667596943360/vN1fmH58_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="157.3" y="264.8">Connor ’Stryxus’ Shearer</tspan></text><text x="66" y="40" class="subText"><tspan x="157.3" y="281.6">@Stryxus</tspan></text></a><a><path fill="url(#a)" d="M550.3 254.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="112.3" y="314">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="112.3" y="339.2">What happens when v8 reaches v8?</tspan></text><image width="22.4" height="20" x="375.6" y="325.2" href="https://abs.twimg.com/emoji/v2/72x72/1f914.png"/></g><a class="like"><path d="M112.3 355.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="132.8" y="369.2">11</tspan></text></a><a><text x="68" y="126" class="subText"><tspan x="159.4" y="369.2">17:19 - 20 Nov 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M383.2 179.6l399.8-14 5.4 126.6-399.8 14z"/><g transform="rotate(-2 1958.9 -3131) scale(.8)"><a><image width="36" height="36" x="356.8" y="174.2" href="https://pbs.twimg.com/profile_images/977905767415205888/Pr5NSZ_T_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="401.8" y="184.8">Thibault Molleman</tspan></text><text x="66" y="40" class="subText"><tspan x="401.8" y="201.6">@thibaultmol</tspan></text></a><a><path fill="url(#a)" d="M794.8 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="356.8" y="234">Replying to @v8js</tspan></text></a><text x="21" y="92"><tspan x="356.8" y="258.4">Wait. What happens when we get V8 V8?</tspan></text><a class="like"><path d="M356.8 274.6h17.5v17.5h-17.5z"/></a><a><text x="54" y="125" class="subText"><tspan x="389.3" y="288.4">11:37 - 20 Jun 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M522 272.1l400-7 2.6 127.4-400 7z"/><g transform="rotate(-1 4619.2 -7976.5) scale(.8)"><a><image width="36" height="36" x="494.3" y="270.2" href="https://pbs.twimg.com/profile_images/1185226911221452802/nH7ykGFP_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="539.3" y="280.8">Greg Miernicki aka 🅖🅡🅔🅖</tspan></text><text x="66" y="40" class="subText"><tspan x="539.3" y="297.6">@gregulatore</tspan></text></a><a><path fill="url(#a)" d="M932.3 270.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="494.3" y="330">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="494.3" y="355.2">Anything special planned for v8 v8.0?</tspan></text><image width="22.4" height="20" x="764.7" y="341.2" href="https://abs.twimg.com/emoji/v2/72x72/1f605.png"/></g><a class="like"><path d="M494.3 371.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="514.8" y="385.2">5</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="534.6" y="385.2">16:43 - 13 Aug 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M671.2 141.3l394 69.5-30 142.7-394-69.5z"/><g transform="rotate(10 469.6 1210) scale(.8)"><a><image width="36" height="36" x="624.2" y="174.2" href="https://pbs.twimg.com/profile_images/22070422/south_park_me__art_raged__normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="669.2" y="184.8">SignpostMarv</tspan></text><text x="66" y="40" class="subText"><tspan x="669.2" y="201.6">@SignpostMarv</tspan></text></a><a><path fill="url(#a)" d="M1062.2 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="624.2" y="234">Replying to @v8js @ChromiumDev</tspan></text></a><text x="21" y="92"><tspan x="624.2" y="258.4">are you going to be having an extra special party when V8 goes</tspan><tspan x="624.2" y="279.4">v8 ?</tspan></text><a class="like"><path d="M624.2 296.6h17.5v17.5h-17.5z"/><text x="42" y="146"><tspan x="644.7" y="310.4">18</tspan></text></a><a><text x="69" y="147" class="subText"><tspan x="672.3" y="310.4">16:20 - 27 Sep 2019</tspan></text></a></g></g></svg>
```

Reply:
Even better:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="1075" height="290" viewBox="-5 140 1075 290" style="position:relative;max-width:90vw;height:auto;margin-top:max(-47px,-5vw);left:50%;margin-left:max(-547px,-45vw)"><style>text{font-family:Helvetica,Roboto,Segoe UI,Calibri,sans-serif;fill:#1c2022;font-weight:400}.bg,.divider{stroke:#e1e8ed;stroke-width:.8;fill:#fff}a.name text{font-weight:700}.seeOther,.subText,a.like text,a.name .subText{fill:#697882;font-size:14px;font-weight:400}.seeOther{fill:#2b7bb9}a.like path{fill:url(#b)}a:hover text{fill:#3b94d9}a.like:hover text{fill:#e0245e}a.like:hover path{fill:url(#B)}.dark .bg{stroke:#66757f;fill:#000}.dark text{fill:#f5f8fa}.dark .subText,.dark a.name .subText,.dark a.like text{fill:#8899a6}.dark .seeOther{fill:#3b94d9}.dark a:hover text{fill:#55acee}.dark a.like:hover text{fill:#e0245e}</style><defs><pattern id="a" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 72 72%22><path fill=%22none%22 d=%22M0 0h72v72H0z%22/><path class=%22icon%22 fill=%22%231da1f2%22 d=%22M68.812 15.14c-2.348 1.04-4.87 1.744-7.52 2.06 2.704-1.62 4.78-4.186 5.757-7.243-2.53 1.5-5.33 2.592-8.314 3.176C56.35 10.59 52.948 9 49.182 9c-7.23 0-13.092 5.86-13.092 13.093 0 1.026.118 2.02.338 2.98C25.543 24.527 15.9 19.318 9.44 11.396c-1.125 1.936-1.77 4.184-1.77 6.58 0 4.543 2.312 8.552 5.824 10.9-2.146-.07-4.165-.658-5.93-1.64-.002.056-.002.11-.002.163 0 6.345 4.513 11.638 10.504 12.84-1.1.298-2.256.457-3.45.457-.845 0-1.666-.078-2.464-.23 1.667 5.2 6.5 8.985 12.23 9.09-4.482 3.51-10.13 5.605-16.26 5.605-1.055 0-2.096-.06-3.122-.184 5.794 3.717 12.676 5.882 20.067 5.882 24.083 0 37.25-19.95 37.25-37.25 0-.565-.013-1.133-.038-1.693 2.558-1.847 4.778-4.15 6.532-6.774z%22/></svg>"/></pattern><pattern id="b" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23697882%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern><pattern id="B" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23E0245E%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern></defs><g data-id="twitter-widget"><path class="bg" d="M-2.2 222.4l398.4-34.8 13.6 127.3-398.4 34.8z"/><g transform="rotate(-5 830.8 -212.3) scale(.8)"><a><image width="36" height="36" x="-25.2" y="206.2" href="https://pbs.twimg.com/profile_images/1204331159582367745/YF3T9NOI_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="19.8" y="218.6">Josebaba</tspan></text><image width="22.4" height="20" x="93.6" y="206.2" href="http://abs.twimg.com/emoji/v2/72x72/1f4a5.png"/><text x="66" y="42" class="subText"><tspan x="19.8" y="235.4">@fullstackmofo</tspan></text></a><a><path fill="url(#a)" d="M412.8 206.2h20v20h-20z"/></a><a><text x="21" y="72" class="subText"><tspan x="-25.2" y="266.8">Replying to @v8js</tspan></text></a><text x="21" y="93"><tspan x="-25.2" y="291.2">V8 almost at v8</tspan></text><a class="like"><path d="M-25.2 307.4h17.5v17.5h-17.5z"/><text x="42" y="125" class="subText"><tspan x="-4.7" y="321.2">4</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="15.1" y="321.2">22:09 - 20 Nov 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M147.2 238.9l399 27.9-10.8 127-399-28z"/><g transform="rotate(4 -638.7 1274.7) scale(.8)"><a><image width="36" height="36" x="112.3" y="254.2" href="https://pbs.twimg.com/profile_images/1205935667596943360/vN1fmH58_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="157.3" y="264.8">Connor ‘Stryxus’ Shearer</tspan></text><text x="66" y="40" class="subText"><tspan x="157.3" y="281.6">@Stryxus</tspan></text></a><a><path fill="url(#a)" d="M550.3 254.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="112.3" y="314">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="112.3" y="339.2">What happens when v8 reaches v8?</tspan></text><image width="22.4" height="20" x="375.6" y="325.2" href="https://abs.twimg.com/emoji/v2/72x72/1f914.png"/></g><a class="like"><path d="M112.3 355.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="132.8" y="369.2">11</tspan></text></a><a><text x="68" y="126" class="subText"><tspan x="159.4" y="369.2">17:19 - 20 Nov 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M383.2 179.6l399.8-14 5.4 126.6-399.8 14z"/><g transform="rotate(-2 1958.9 -3131) scale(.8)"><a><image width="36" height="36" x="356.8" y="174.2" href="https://pbs.twimg.com/profile_images/977905767415205888/Pr5NSZ_T_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="401.8" y="184.8">Thibault Molleman</tspan></text><text x="66" y="40" class="subText"><tspan x="401.8" y="201.6">@thibaultmol</tspan></text></a><a><path fill="url(#a)" d="M794.8 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="356.8" y="234">Replying to @v8js</tspan></text></a><text x="21" y="92"><tspan x="356.8" y="258.4">Wait. What happens when we get V8 V8?</tspan></text><a class="like"><path d="M356.8 274.6h17.5v17.5h-17.5z"/></a><a><text x="54" y="125" class="subText"><tspan x="389.3" y="288.4">11:37 - 20 Jun 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M522 272.1l400-7 2.6 127.4-400 7z"/><g transform="rotate(-1 4619.2 -7976.5) scale(.8)"><a><image width="36" height="36" x="494.3" y="270.2" href="https://pbs.twimg.com/profile_images/1185226911221452802/nH7ykGFP_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="539.3" y="280.8">Greg Miernicki</tspan></text><text x="66" y="40" class="subText"><tspan x="539.3" y="297.6">@gregulatore</tspan></text></a><a><path fill="url(#a)" d="M932.3 270.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="494.3" y="330">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="494.3" y="355.2">Anything special planned for v8 v8.0?</tspan></text><image width="22.4" height="20" x="764.7" y="341.2" href="https://abs.twimg.com/emoji/v2/72x72/1f605.png"/></g><a class="like"><path d="M494.3 371.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="514.8" y="385.2">5</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="534.6" y="385.2">16:43 - 13 Aug 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M671.2 141.3l394 69.5-30 142.7-394-69.5z"/><g transform="rotate(10 469.6 1210) scale(.8)"><a><image width="36" height="36" x="624.2" y="174.2" href="https://pbs.twimg.com/profile_images/22070422/south_park_me__art_raged__normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="669.2" y="184.8">SignpostMarv</tspan></text><text x="66" y="40" class="subText"><tspan x="669.2" y="201.6">@SignpostMarv</tspan></text></a><a><path fill="url(#a)" d="M1062.2 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="624.2" y="234">Replying to @v8js @ChromiumDev</tspan></text></a><text x="21" y="92"><tspan x="624.2" y="258.4">are you going to be having an extra special party when V8 goes</tspan><tspan x="624.2" y="279.4">v8 ?</tspan></text><a class="like"><path d="M624.2 296.6h17.5v17.5h-17.5z"/><text x="42" y="146"><tspan x="644.7" y="310.4">18</tspan></text></a><a><text x="69" y="147" class="subText"><tspan x="672.3" y="310.4">16:20 - 27 Sep 2019</tspan></text></a></g></g></svg>
```

Reply:
Wow, you really hate those bottom bars ;) Done.

Reply:
Best:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="1075" height="290" viewBox="-5 140 1075 290" style="position:relative;max-width:90vw;height:auto;margin-top:max(-47px,-5vw);left:50%;margin-left:max(-547px,-45vw)"><style>text{font-family:Helvetica,Roboto,Segoe UI,Calibri,sans-serif;fill:#1c2022;font-weight:400}.bg{stroke:#e1e8ed;stroke-width:.8;fill:#fff}a.name text{font-weight:700}.subText,a.like text,a.name .subText{fill:#697882;font-size:14px;font-weight:400}a.like path{fill:url(#b)}a:hover text{fill:#3b94d9}a.like:hover text{fill:#e0245e}a.like:hover path{fill:url(#B)}</style><defs><pattern id="a" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 72 72%22><path fill=%22none%22 d=%22M0 0h72v72H0z%22/><path class=%22icon%22 fill=%22%231da1f2%22 d=%22M68.812 15.14c-2.348 1.04-4.87 1.744-7.52 2.06 2.704-1.62 4.78-4.186 5.757-7.243-2.53 1.5-5.33 2.592-8.314 3.176C56.35 10.59 52.948 9 49.182 9c-7.23 0-13.092 5.86-13.092 13.093 0 1.026.118 2.02.338 2.98C25.543 24.527 15.9 19.318 9.44 11.396c-1.125 1.936-1.77 4.184-1.77 6.58 0 4.543 2.312 8.552 5.824 10.9-2.146-.07-4.165-.658-5.93-1.64-.002.056-.002.11-.002.163 0 6.345 4.513 11.638 10.504 12.84-1.1.298-2.256.457-3.45.457-.845 0-1.666-.078-2.464-.23 1.667 5.2 6.5 8.985 12.23 9.09-4.482 3.51-10.13 5.605-16.26 5.605-1.055 0-2.096-.06-3.122-.184 5.794 3.717 12.676 5.882 20.067 5.882 24.083 0 37.25-19.95 37.25-37.25 0-.565-.013-1.133-.038-1.693 2.558-1.847 4.778-4.15 6.532-6.774z%22/></svg>"/></pattern><pattern id="b" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23697882%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern><pattern id="B" width="1" height="1" patternContentUnits="objectBoundingBox" patternUnits="objectBoundingBox"><image width="1" height="1" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 width=%2224%22 height=%2224%22 viewBox=%220 0 24 24%22><path class=%22icon%22 fill=%22%23E0245E%22 d=%22M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.813-1.148 2.353-2.73 4.644-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.375-7.454 13.11-10.037 13.156H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.035 11.596 8.55 11.658 1.52-.062 8.55-5.917 8.55-11.658 0-2.267-1.822-4.255-3.902-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.015-.03-1.426-2.965-3.955-2.965z%22/></svg>"/></pattern></defs><g data-id="twitter-widget"><path class="bg" d="M-2.2 222.4l398.4-34.8 13.6 127.3-398.4 34.8z"/><g transform="rotate(-5 830.8 -212.3) scale(.8)"><a><image width="36" height="36" x="-25.2" y="206.2" href="https://pbs.twimg.com/profile_images/1204331159582367745/YF3T9NOI_normal.jpg"/></a><a class="name"><text x="94" y="21"><tspan x="19.8" y="218.6">Josebaba 💥</tspan></text><text x="66" y="42" class="subText"><tspan x="19.8" y="235.4">@fullstackmofo</tspan></text></a><a><path fill="url(#a)" d="M412.8 206.2h20v20h-20z"/></a><a><text x="21" y="72" class="subText"><tspan x="-25.2" y="266.8">Replying to @v8js</tspan></text></a><text x="21" y="93"><tspan x="-25.2" y="291.2">V8 almost at v8</tspan></text><a class="like"><path d="M-25.2 307.4h17.5v17.5h-17.5z"/><text x="42" y="125" class="subText"><tspan x="-4.7" y="321.2">4</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="15.1" y="321.2">22:09 - 20 Nov 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M147.2 238.9l399 27.9-10.8 127-399-28z"/><g transform="rotate(4 -638.7 1274.7) scale(.8)"><a><image width="36" height="36" x="112.3" y="254.2" href="https://pbs.twimg.com/profile_images/1205935667596943360/vN1fmH58_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="157.3" y="264.8">Connor ‘Stryxus’ Shearer</tspan></text><text x="66" y="40" class="subText"><tspan x="157.3" y="281.6">@Stryxus</tspan></text></a><a><path fill="url(#a)" d="M550.3 254.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="112.3" y="314">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="112.3" y="339.2">What happens when V8 reaches v8?</tspan></text><image width="22.4" height="20" x="375.6" y="325.2" href="https://abs.twimg.com/emoji/v2/72x72/1f914.png"/></g><a class="like"><path d="M112.3 355.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="132.8" y="369.2">11</tspan></text></a><a><text x="68" y="126" class="subText"><tspan x="159.4" y="369.2">17:19 - 20 Nov 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M383.2 179.6l399.8-14 5.4 126.6-399.8 14z"/><g transform="rotate(-2 1958.9 -3131) scale(.8)"><a><image width="36" height="36" x="356.8" y="174.2" href="https://pbs.twimg.com/profile_images/977905767415205888/Pr5NSZ_T_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="401.8" y="184.8">Thibault Molleman</tspan></text><text x="66" y="40" class="subText"><tspan x="401.8" y="201.6">@thibaultmol</tspan></text></a><a><path fill="url(#a)" d="M794.8 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="356.8" y="234">Replying to @v8js</tspan></text></a><text x="21" y="92"><tspan x="356.8" y="258.4">Wait. What happens when we get V8 v8?</tspan></text><a class="like"><path d="M356.8 274.6h17.5v17.5h-17.5z"/></a><a><text x="54" y="125" class="subText"><tspan x="389.3" y="288.4">11:37 - 20 Jun 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M522 272.1l400-7 2.6 127.4-400 7z"/><g transform="rotate(-1 4619.2 -7976.5) scale(.8)"><a><image width="36" height="36" x="494.3" y="270.2" href="https://pbs.twimg.com/profile_images/1185226911221452802/nH7ykGFP_normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="539.3" y="280.8">Greg Miernicki</tspan></text><text x="66" y="40" class="subText"><tspan x="539.3" y="297.6">@gregulatore</tspan></text></a><a><path fill="url(#a)" d="M932.3 270.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="494.3" y="330">Replying to @v8js</tspan></text></a><g data-id="p"><text x="21" y="92"><tspan x="494.3" y="355.2">Anything special planned for V8 v8.0?</tspan></text><image width="22.4" height="20" x="764.7" y="341.2" href="https://abs.twimg.com/emoji/v2/72x72/1f605.png"/></g><a class="like"><path d="M494.3 371.4h17.5v17.5h-17.5z"/><text x="42" y="125"><tspan x="514.8" y="385.2">5</tspan></text></a><a><text x="61" y="126" class="subText"><tspan x="534.6" y="385.2">16:43 - 13 Aug 2019</tspan></text></a></g></g><g data-id="twitter-widget"><path class="bg" d="M671.2 141.3l394 69.5-30 142.7-394-69.5z"/><g transform="rotate(10 469.6 1210) scale(.8)"><a><image width="36" height="36" x="624.2" y="174.2" href="https://pbs.twimg.com/profile_images/22070422/south_park_me__art_raged__normal.jpg"/></a><a class="name"><text x="66" y="21"><tspan x="669.2" y="184.8">SignpostMarv</tspan></text><text x="66" y="40" class="subText"><tspan x="669.2" y="201.6">@SignpostMarv</tspan></text></a><a><path fill="url(#a)" d="M1062.2 174.2h20v20h-20z"/></a><a><text x="21" y="71" class="subText"><tspan x="624.2" y="234">Replying to @v8js @ChromiumDev</tspan></text></a><text x="21" y="92"><tspan x="624.2" y="258.4">are you going to be having an extra special party when V8 goes</tspan><tspan x="624.2" y="279.4">v8 ?</tspan></text><a class="like"><path d="M624.2 296.6h17.5v17.5h-17.5z"/><text x="42" y="146"><tspan x="644.7" y="310.4">18</tspan></text></a><a><text x="69" y="147" class="subText"><tspan x="672.3" y="310.4">16:20 - 27 Sep 2019</tspan></text></a></g></g></svg>
```